annotation-file-utilities/src/annotator/find/InheritedSymbolFinder.java

```diff
@@ -0,0 +1,211 @@
+package annotator.find;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.util.ElementFilter;
+import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
+
+import annotations.util.JVMNames;
+import annotator.Source;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.LinkedHashMultimap;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.Multimaps;
+
+import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.model.JavacElements;
+
+/**
+ * Provides static access to JVML descriptors of methods for which
```

Comment:
What is "static access"?  It would be clearer to say that this class doesn't represent anything and is just a collection of static methods.  Furthermore, add a private constructor that throws an exception, so clients can't accidentally instantiate it.